.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description (optional)
+
+-
+
+## Issue that this pull request resolves
+
+Closes: mozilla/experimenter#[0000]
```

Comment:
There's an explanation in `contributing.md`, but I wonder if I should add a note here about changing the repository name if needed? Also I think I'll take out the brackets so someone doesn't leave those in thinking they're needed.

Reply:
Yeah, I would at least add a comment here about removing it / what it's for, or else leave it out and describe in the docs how to add. I'd guess a lot of folks who are doing this will be using git/the commit message anyway (although maybe not)?

Reply:
Updated it so it's more clear👍